test/broker/src/test/app-utils/test-app/app.js

```diff
@@ -39,16 +37,18 @@ app.post('/usage', (req, res) => {
     body: usage
   }, (err, response) => {
     if (err) {
-      console.log('An error posting usage occured: %o', err);
+      console.log('An error posting usage occured: %s', err);
       res.status(500).send(err);
       return;
     }
-    console.log('Collector replied with %s and %s, %o',response.statusCode,
+    console.log('Collector replied with %s and %s, %s',response.statusCode,
       response.message, response.body);
     res.status(response.statusCode).send(response.message);
   });
 });
 
-app.listen(PORT, function() {
-  console.log('Server running on http://localhost:' + PORT);
+usageToken.start(() => {
+  app.listen(port, () => {
+    console.log('Server running on http://localhost:' + port);
```

Comment:
Do we need console.log at all? Should this be debug instead?

Reply:
Yes, this is a sample app used only for the tests, so we haven't added the debug module and a separate env variable. From my POV console.log is sufficient for debugging the app in case of failures.